docs/products/access/api-reference/config-api/service-providers-api.md

```diff
@@ -0,0 +1,333 @@
+| signing_certificate                 | yes      | "-----BEGIN CERTIFICATE-----\r\nMIIEnTCCAoC ..." | Plain certificate for assertions signing in x509 format.                                                 |
+| encryption_certificate              | no       | "-----BEGIN CERTIFICATE-----\r\nMIIEnTCCAoC ..." | Plain certificate for encryption in x509 format.                                                         |
+| logout_url_redirect                 | no       | "https://example.sp.com/logout"                  | Location of the Single Logout Service (POST binding).                                                    |
+| logout_url_post                     | no       | "https://example.sp.com/logout_post"             | Location of the Single Logout Service (Redirect binding).                                                |
+
+#### Limitations
+
+1. Entity ID must be unique in the scope of one tenant.
+2. The `identity_provider` and `backup_identity_providers` fields are references to other parts in the configuration of Access.
+   This API can only refer to existing identifiers of Identity Providers. When an unknown identifier is passed, the response will contain
+   a validation error: `Identity provider [provided_id] not found`.
+3. In the case of `URL` and `XML` metadata type, when `validUntil` attribute is specified in Entity Descriptor, Access will check the
+   metadata
+   validity and return an error in case of expired metadata.
+4. In the case of `URL` and `XML` metadata type, when the xml metadata is signed, Access will check the signature validity and return an
+   error in case of failed verification.
+5. Provided metadata must contain `SPSSODescriptor` element, containing a signing certificate, and at least one `AssertionConsumerService`
+6. The max length of `name` and `entity_id` fields is 255.
+
+Example request:
+
+```http
+POST /api/v1/saml-sp-config HTTP/1.1
+Host: onewelcome.example.com
+Content-Type: application/json
+{
+    "name": "My Service Provider",
+    "metadata_type": "MANUAL",
+    "manual_metadata": {
+        "entity_id": "myEntityId",
+        "assertion_consumer_service_binding": "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST",
+        "assertion_consumer_service_location": "https://example.sp.com/authn-response",
+        "signing_certificate": "-----BEGIN CERTIFICATE----- (...) -----END CERTIFICATE-----",
+        "logout_url_post": "https://example.sp.com/logout"
+    },
+    "user_identifier": "Email",
+    "attribute_mappings": {
+        "email": "Email address",
+        "display_name": "Display name"
+    },
+    "identity_provider": "oidc_idp"
+}
+```
+
+Example success response:
+
+```http
+HTTP/1.1 201 CREATED
+Content-Type: application/json;charset=UTF-8
+Cache-Control: no-store
+Pragma: no-cache
```

Comment:
I think we should return Location header

Reply:
I created a PR for that: https://github.com/onewelcome/access/pull/2024